htlcswitch/switch.go

```diff
@@ -723,9 +723,10 @@ func (s *Switch) htlcForwarder() {
 
 			// Otherwise, we'll log this diff, then accumulate the
 			// new stats into the running total.
-			log.Infof("Sent %v satoshis received %v satoshi "+
+			log.Infof("Sent %v satoshis received %v satoshis "+
 				"in the last 10 seconds (%v tx/sec)",
```

Comment:
`float64(diffNumUpdates)/10` could have many a few decimal places. Is there any thought of capping the number of decimals when logging?